tests/acceptance/features/bootstrap/SharingNgContext.php

```diff
@@ -104,6 +104,9 @@ public function userSendsTheFollowingShareInvitationUsingTheGraphApi(string $use
 			? $this->featureContext->getAttributeOfCreatedUser($rows['sharee'], 'id')
 			: $this->featureContext->getAttributeOfCreatedGroup($rows['sharee'], 'id');
 
+		$role = $rows['role'] ?? null;
+		$permission = $rows['permission'] ?? null;
```

Comment:
```suggestion
		$role = $rows['role'] ?: null;
		$permission = $rows['permission'] ?: null;
```

Reply:
Using ternary op will be a problem when either role or permission is passed from the data table.

Reply:
it did not work for me!

Reply:
> not

which one??

Reply:
> it did not work for me!

which one??

Reply:
what will be the error?
It just because `??` checks null and we are assigning null

Reply:
I think, there will be key doesn't exist error. maybe it is okey